src/formats/png/crc.zig

```diff
@@ -0,0 +1,87 @@
+const std = @import("std");
+const io = std.io;
+
+const crc = @This();
+
+var crc_table: ?[256]u32 = null;
```

Comment:
This can be `const` and computed at compile-time

```zig
const crc_table:[256]u32 = blk: {
   const magic_n = 0xedb88320;

    var crc_table:[256]u32 = undefined;

    var n: u32 = 0;
    while (n < 256) : (n += 1) {
        var c: u32 = n;
        var k: usize = 0;
        while (k < 8) : (k += 1) {
            if (c & 1 != 0) {
                c = magic_n ^ (c >> 1);
            } else {
                c >>= 1;
            }
        }
        crc_table[n] = c;
    }
    
    break :blk crc_table;
};
```